app/src/lib/feature-flag.ts

```diff
@@ -133,3 +133,10 @@ export function enableForkSettings(): boolean {
 export function enableDiscardLines(): boolean {
   return true
 }
+
+/**
+ * Should we allow to change the default branch when creating new repositories?
+ */
+export function enableDefaultBranchSetting(): boolean {
+  return enableDevelopmentFeatures()
```

Comment:
As discussed in the async sync today we can ship this to beta now.

```suggestion
  return enableBetaFeatures()
```